### PR TITLE
Update storybook to v6

### PR DESCRIPTION
**Pull request by matthewroach**

### CODE REVIEW

**Comment by JackHowa** [comment]
excited for this ... will review

File: .storybook/.babelrc [hunk 1]
```diff
@@ -1,10 +1,6 @@
 {
   "plugins": [
-    [
-      "@babel/plugin-proposal-decorators",
-      {
-        "legacy": true
-      }
-    ]
+    ["@babel/plugin-proposal-private-property-in-object", { "loose": true }],
```
Comment (JackHowa) [inline .storybook/.babelrc:3]: did you add this plugin?

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "lint:styles": "stylelint '**/*.(scss|jsx)' --formatter verbose",
     "lint:styles:fix": "stylelint '**/*.scss' --fix",
     "lint:fix": "npm run lint -- --fix",
-    "storybook": "start-storybook",
+    "storybook": "start-storybook -p 60001",
```
Comment (JackHowa) [inline package.json:18]: why does it need to be on this port?

**Review by JackHowa — DISMISSED** [review]
works as expected! looks great

**Comment by JackHowa** [comment]
closes https://github.com/WPMedia/fusion-news-theme-blocks/issues/487

**Comment by JackHowa** [comment]
looks like there's some substantive differences in the review stage? https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=6098f4a784ad4f003bcdae8e old diff?

File: package.json (hunk 2 above)
Comment (matthewroach) [reply package.json:18]: Not needed per say, but as we have more storybooks having them on a specific port allows better linking

File: .storybook/.babelrc (hunk 1 above)
Comment (matthewroach) [reply .storybook/.babelrc:3]: Starting storybook threw a load of warnings suggesting to add this.

File: package.json (hunk 2 above)
Comment (JackHowa) [reply package.json:18]: gotcha no worries, thought it may have been accidental

**Comment by matthewroach** [comment]
Not sure about the chromatic tests as they shouldn't have changed

File: .storybook/.babelrc (hunk 1 above)
Comment (JackHowa) [reply .storybook/.babelrc:3]: huh interesting we didn't need to add it to package.json. I guess it must be a nested dep

**Comment by matthewroach** [comment]
Fixed the storybook layout issues @JackHowa in v6 they have a layout parameter that has a different default to pre v6 - https://storybook.js.org/docs/react/configure/story-layout

**Comment by JackHowa** [comment]
> Fixed the storybook layout issues @JackHowa in v6 they have a layout parameter that has a different default to pre v6 - https://storybook.js.org/docs/react/configure/story-layout

nice good find. didn't know about this one

**Review by JackHowa — APPROVED** [review]
good work! excited this is working

**Pull request merged**